app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CommentsController < ApplicationController
+  def create
+    @article = Article.find(params[:article_id])
+    @comment = @article.comments.create(comment_params)
```

Comment:
Its unnecesary to save a variable if you are not using it afterwards in the view.
Also, what would you do if the create method fails? You should do something about it, show an error in the view.
(Search about model errors)